test/processes_mysql.jl

```diff
@@ -51,46 +52,89 @@ end
 @testset "MESH2UMLS" begin
     # $TRAVIS is equal to "true" if it's a Travis build, and "false" if it's not.
     is_travis = lowercase(strip(get(ENV, "TRAVIS", ""))) == "true"
+
     # $TRAVIS_PULL_REQUEST is equal to the PR number if it is a PR, and "false" if it's not.
     is_travis_pull_request = lowercase(strip(get(ENV, "TRAVIS_PULL_REQUEST", ""))) != "false"
+
     # If this is a Travis build, only execute this test if it is NOT a pull request.
     # If this is not a Travis build, then always execute this test.
     if !is_travis || !is_travis_pull_request
         println("-----------------------------------------")
         println("       Testing MESH2UMLS")
         append = false
-        @time begin
-            map_mesh_to_umls_async!(conn, umls_user, umls_pswd; append_results=append, timeout=1)
+
+        success = try
```

Comment:
The variable for success approach is good, but instead of setting `success = try ... catch ... end` could you initialize `success = true` and then in the `catch` set it to `false`.  While it is the same result, it makes the code a bit easier to read.

Reply:
Done!